test/e2e/certs.go

```diff
@@ -25,18 +25,19 @@ import (
 )
 
 var _ = Describe("MasterCerts", func() {
+	supportedProviders := []string{"gce", "gke"}
+
 	BeforeEach(func() {
 		var err error
 		_, err = loadClient()
 		Expect(err).NotTo(HaveOccurred())
-	})
 
-	It("should have all expected certs on the master", func() {
-		if !providerIs("gce", "gke") {
-			By(fmt.Sprintf("Skipping MasterCerts test for cloud provider %s (only supported for gce and gke)", testContext.Provider))
-			return
+		if !providerIs(supportedProviders...) {
+			Skipf("Only supported for providers %v (not %s)", supportedProviders, testContext.Provider)
```

Comment:
this pattern is common enough that I wonder if we should have a wrapper which calls `providerIs()` followed by `Skipf` if needed.